### PR TITLE
Training card guide

### PR DESCRIPTION
**Pull request by latin-panda**
# Description

Technical guide for training cards.

medic/cht-core#6598

[CHT-Conf PR](https://github.com/medic/cht-conf/pull/529) - Command line to convert and upload training forms
[CHT-Core PR](https://github.com/medic/cht-core/pull/8017) - Feature

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
Hi @esthermmoturi @n-orlowski @ngaruko please review this content for training cards, thanks!

**Review by n-orlowski — APPROVED** [review]
LGTM! Do we need an updated asset for the training deck png?

File: content/en/apps/guides/training/training-cards/_index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  Deploy in-app training cards for remote training.
+keyword: training
+relatedContent: >
+  apps/features/training
+  apps/examples/training
+  apps/examples/learning-care/
+---
+_Introduced in 4.2.0_
+
+[Training Cards]({{< relref "apps/features/training" >}}) enable remote training from within the CHT by showing a sequence of "cards" containing content provided by App Developers. Enketo forms are used to display the content, and App Developers can specify a start date, duration, and to which [user roles]({{< relref "apps/reference/app-settings/user-roles" >}}) the cards should be shown. Like [app forms]({{< relref "apps/reference/forms/app" >}}), forms used by training cards will automatically be downloaded to the user’s devices.
```
Comment (esthermmoturi) [inline content/en/apps/guides/training/training-cards/_index.md:15]: Is it possible to include a short description of the kind of content that will be provided or the context of training cards. E.g that they explain changes in features or UI/UX

File: content/en/apps/guides/training/training-cards/_index.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+In the example above, the training cards could be shown to any user with the "nurse" role between July 13, 2023 and September 11, 2023 (inclusive). See more information about these configuration settings below: 
+| Property | Description |
+|---|----|
+| "title" | Optional. Enketo’s form title that is displayed under the modal’s header section. Leave this property empty if you don’t need to display a title. |
+| "start_date" | Optional. Define the training start day using `yyyy-mm-dd` format. If not defined then the training will start immediately. |
+| "duration" | Optional. Number of days this training should be active. If not defined then the training will never expire. |
+| "user_roles" | Optional. List of user roles that can access this training. If not defined then all users can access the training. |
+
+{{% alert title="Note" %}} Users with role admin can access training cards, but they need to have a contact associated in the `org.couchdb.user:[user-name]` document from CouchDB. {{% /alert %}}
```
Comment (esthermmoturi) [inline content/en/apps/guides/training/training-cards/_index.md:56]:
```suggestion
{{% alert title="Note" %}} Users with an admin role can access training cards but they need to have a contact associated in the `org.couchdb.user:[user-name]` document from CouchDB. {{% /alert %}}
```

Comment (latin-panda) [reply content/en/apps/guides/training/training-cards/_index.md:56]: Thanks! Done

File: content/en/apps/guides/training/training-cards/_index.md (hunk 1 above)
Comment (latin-panda) [reply content/en/apps/guides/training/training-cards/_index.md]: Good idea! I've added a short description.

**Comment by latin-panda** [comment]
@esthermmoturi thanks for your feedback, can you please have a look again? 

@michaelkohn would you like to change the image for the training deck png?

**Review by esthermmoturi — APPROVED** [review]

**Comment by michaelkohn** [comment]
> @michaelkohn would you like to change the image for the training deck png?

@latin-panda during our CT call, we agreed that we need to get rid of the phone outline / overlay and that you would be able to take the screenshots using the real feature. You probably have what you need already since we deployed these cards before, but [here's the content](https://drive.google.com/drive/u/0/folders/1hLwx-UuzsQZE03LJpIR_Gn0gCFW8qj-F) we used for them just in case it's useful!

**Comment by latin-panda** [comment]
Very helpful, thanks for the link @michaelkohn! 
I've uploaded the new image, here is how it looks in the docs site.
Screenshot 2023-03-08 at 1 04 23 pm

**Comment by michaelkohn** [comment]
Awesome, thanks @latin-panda ! Just like a Picasso 

1943-original-picasso-femme-dans-un-fauteuil-lithograph-7268

**Pull request merged**